### ISSUE TITLE
Better MC [Forge] *SERVER* v30.5 Random crashes /w solution

### ISSUE THREAD

**Issue by Acidburn0x90** [LunaPixelStudios/Better-MC#1799]
### Random Crash 

Whenever I would launch this sever I would experience random crashes with no apparent cause. Whenever this would happen it would not allow any terminal commands, ctrl+c, or text input: completely locking up the system. I had this issue and and decided to let it run for as long as it needed, and as it ran overnight, it finally produced a crash log. My solution for the random crashes was to remove the mod Neruina. WIth future updates to the modpack and the mod itself, this problem may resolve on its own.

TL;DR:
Remove mod Neruina

System specifications:
Raspberry Pi5 8GB running Ubuntu Server 24.04 LTS

### Reproduction Steps

1. Start server with the ./start.sh
2. Have server run  for approx ~15 minutes with 3-5 players
3. Crash

### Operating System

Ubuntu Server 20.04 LTS

### Minecraft Version

1.20.1

### Modloader

Forge

### Modpack Version

v30.5
